cloud_formation/configs/core.py

```diff
@@ -389,10 +390,11 @@ def post_init(bosslet_config):
         print("Creating initial Keycloak admin user")
         # This fails if the user already exists, but execution will continue.
         ssh("/srv/keycloak/bin/add-user-keycloak.sh -r master -u {} -p {}".format(username, password))
+        time.sleep(10)
 
         print("Restarting Keycloak")
         ssh("sudo service keycloak stop")
-        time.sleep(2)
+        time.sleep(3)
         ssh("sudo killall java") # the daemon command used by the keycloak service doesn't play well with standalone.sh
```

Comment:
You found that 'ssh("/srv/keycloak/bin/jboss-cli.sh --connect reload")` definitely doesn't work properly for setup? `killall` will have to be installed for the newest Ubuntu 20 AMI.

Reply:
Yeah it didn't actually restart.  Nothing showed up in the server.log when I did that way

Reply:
Merged in @movestill changes from branch restart-keycloak-with-cli